R/java_env.R

```diff
@@ -248,13 +255,13 @@ java_check_version_cmd <- function(
   # Get JAVA_HOME again and check if it's set
   current_java_home <- Sys.getenv("JAVA_HOME")
   if (current_java_home == "") {
-    cli::cli_alert_warning("JAVA_HOME is not set.")
+    if (!quiet) cli::cli_alert_warning("JAVA_HOME is not set.")
```

Comment:
FWIW we've mostly switched to this style: `cli::cli_inform(c("!" = "JAVA_HOME is not set."))`